app/views/hyrax/admin/admin_sets/_form_workflow.erb

```diff
@@ -0,0 +1,27 @@
+<% # OVERRIDE: Hyrax 3.5.0 to remove the hyku_commons_mediated_deposit workflow %>
+<div id="workflow" class="tab-pane">
+<div class="panel panel-default labels">
+  <%= simple_form_for collection_permission_template_form_for(form: @form),
+    url: [hyrax, :admin, @form, :permission_template],
+    html: { id: 'form_workflows', class: 'nav-safety' } do |f| %>
+    <% if f.object.available_workflows.any? %>
+      <div class="panel-body">
+        <p><%= t('.page_description') %></p>
+            <% # OVERRIDE: Hyrax 3.5.0 to reject the hyku_commons_mediated_deposit workflow %>
+            <%= f.collection_radio_buttons :workflow_id, f.object.available_workflows.reject { |workflow| workflow.name == 'hyku_commons_mediated_deposit' }, :id, :label, item_wrapper_tag: :div, item_wrapper_class: "radio" do |b| %>
```

Comment:
just curious, had you tried updating `f.object.available_workflows` to not include "hyku_commons_mediated_deposit"?

Reply:
.available_workflows is not a method on admin_sets, just admin_sets.permission_template.available_workflows.

Reply:
I am having a hard time visualizing this, can you give me an example?

Reply:
Find where `available_workflows` is defined and it will help you figure out how to remove it. This kind of logic shouldn't live in the view files

Reply:
try here: `config/workflows`

Reply:
`available_workflows` is defined in Hyrax. It is all of the `Sipity::Workflow` that belong to the `PermissionTemplate`
I'm not sure if it's an object that is saved in the database or not.

Reply:
Did they want to remove the whole workflow? I was under the impression that they only wanted the workflow removed form the admin_set. The available workflow's for admin_set is at admin_set.permission_template.available_workflows, when Sipity::Workflow.activate!(permission_template: admin_set.permission_template.id, workflow_name: 'one_step_mediated_deposit') it activates the active_workflow. So I should remove the filter from the view, add it to the controller and then alter the method in the view?

Reply:
Thank you for your comments Lea Ann I was able to speak with Crystal and removing in entirely is desirable. And much easier than filtering in the view.